docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,94 @@ the latest changes, and propose to you to merge the changes, so you can run it o
 
 ## Upgrading to Volto 16.x.x
 
+### `volto-slate` is now core
+
+From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled as the default text block.
+The previous text block `text` based on `draftJS` is now deprecated and is restricted (hidden) in Volto bootstrap.
+This is a major change and should be planned in advance before you install 16.0.0-alpha.15 or later.
+
+```{versionadded} 16.0.0-alpha.15
+`volto-slate` added to Volto core as the default text block.
+```
+
+```{deprecated} 16.0.0-alpha.15
+`text` text block based on `draftJS` is now deprecated and will be removed from core in Volto 18.
+```
```

Comment:
Is there a roadmap for removal of `draftJS` in a future version of Plone, such as Plone 6.2 or 7.0, or is it "whatever version number of Plone is at whenever we release Volto 18"?

Reply:
See https://github.com/plone/volto/pull/3529